### ISSUE TITLE
PC with nvidia 610 on manjaro

### ISSUE THREAD

**Issue by legion1978** [Bumblebee-Project/Bumblebee#617]
hello
ill try to be clear as i have not much experience/success handling xorg-related (?) issues.

i keep seeing this bumblebee driver supposed to be for laptops? ive had my PCs with nvidia cards for some time with no [perceivable] issues (that i know of).. then i update manjaro (clean install 0.8.11 Ascella 32) and... its just wrong.

i remember having trouble with my previous card (also nvidia) and then i recalled having to replace the video drivers from the system settings manager. no big deal (except for the BIOS switch at startup to enable pci after the driver was installed/replaced) and it did-does work. 

however, unlike before, the computer behaves choppy.. gets stuck playing videos and runs out of ram quickly (4gb) and makes almost no use of swap. its not [or shouldnt be] using the integrated chip (set on BIOS to use only 64mb of ram) so it shouldnt need it anyway (the card has 2gb dedicated memory)

then i see the machine is using the nouveau-bumblebee instead the nvidia-bumblebee.. but when i try to swap.. man.. i cant make it work. it wont let me uninstall nouveau cause of mesa, then i remove mesa and install nvidia along with mesa, but upon restart it just wont. says something about emergency mode with no graphics (only tty). so i tried manually updating/installing mhwd configs to no avail. 

even after reinstalling [the system -for the10th time] it wont boot up with graphics, since "theres no suitable _DMS call" .. whatever that means. 

so, to sum up: i would be happy to use the nouveau drivers, if there were no ram issue, or switch to nvidia if it didnt break the graphics upon restart. but i have no f idea how to accomplish either (after 2 days of google diggin + trial and error)

any help will be appreciated :)
just ask if you guys need more information 

thanks in advance
